### PR TITLE
Update to Nuxt 3.8

### PR DESCRIPTION
**Pull request by walsh9**
- Update to Nuxt 3.8
- Update to Node 18
- Remove vitest from dependencies (until we add some vitest tests)
- Remove a bunch of other dependencies we no longer use or are already included

### CODE REVIEW

**Review by kimlarocca — APPROVED** [review]
💪💪💪

File: .circleci/config.yml [hunk 1]
```diff
@@ -125,7 +125,7 @@ jobs:
 
   test_nginx:
     docker:
-      - image: node:16.14.2-slim
+      - image: node:18-slim
```
Comment (skacula) [inline .circleci/config.yml:128]: Please update the image to use the following version: 18.18.2-slim

File: Dockerfile [hunk 2]
```diff
@@ -33,21 +33,21 @@ RUN npm install sass
 COPY . .
 RUN npm run build
 
-FROM node:16.14.2-slim as app
+FROM node:18-slim as app
```
Comment (skacula) [inline Dockerfile:36]:
```suggestion
FROM node:18.18.2-slim as app
```

File: Dockerfile [hunk 3]
```diff
@@ -1,4 +1,4 @@
-FROM node:16.14.2 as build
+FROM node:18 as build
```
Comment (skacula) [inline Dockerfile:1]:
```suggestion
FROM node:18.18.2 as build
```

File: .circleci/config.yml (hunk 1 above)
Comment (skacula) [inline .circleci/config.yml:128]:
```suggestion
      - image: node:18.18.2-slim
```

File: squash_Dockerfile [hunk 4]
```diff
@@ -1,4 +1,4 @@
-FROM node:16.14.2 as build
+FROM node:18 as build
```
Comment (skacula) [inline squash_Dockerfile:1]:
```suggestion
FROM node:18.18.2 as build
```

File: squash_Dockerfile [hunk 5]
```diff
@@ -16,7 +16,7 @@ RUN npm install sass
 COPY . .
 RUN npm run build
 
-FROM node:16.14.2-slim as app
+FROM node:18-slim as app
```
Comment (skacula) [inline squash_Dockerfile:19]:
```suggestion
FROM node:18.18.2-slim as app
```

**Review by skacula — CHANGES_REQUESTED** [review]
I've added suggestions to change the version to a pinned version. This is the version being used for wnyc-vue3 project.

**Review by skacula — APPROVED** [review]
LGTM!

**Pull request merged**